### PR TITLE
Added the feature to edit images in landscape mode

### LINKED ISSUE #1615: Add Editing feature in Landscape mode
**Actual Behaviour**

Currently on rotating a device when we are editing a photo nothing happens.
**Expected Behaviour**

I want to make a feature that allows the app to edit even when in landscape mode.

**Steps to reproduce it**

Open any image from gallery, click on the edit button and then rotate the device.

**Would you like to work on the issue?**

Yes, definitely

### PR DESCRIPTION
**Pull request by batbrain7**
Fix #1615 

Changes: 
Added the feature to edit images even when in landscape mode.

Screenshots for the change: 
screenshot_20180108-172846
screenshot_20180108-172920
screenshot_20180109-155038

### CODE REVIEW

**Comment by batbrain7** [comment]
@isuruAb review this

**Comment by batbrain7** [comment]
@mohitmanuja @anantprsd5 please review

File: app/src/main/java/org/fossasia/phimpme/editor/EditImageActivity.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public void onCreate(Bundle savedInstanceState) {
         if (getSupportActionBar() != null)
             getSupportActionBar().hide();
 
-        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+//        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/editor/EditImageActivity.java]: Remove the commented lines.

**Comment by batbrain7** [comment]
@anantprsd5 made the changes

**Comment by batbrain7** [comment]
@isuruAb  @anantprsd5  please review and merge this PR

**Comment by batbrain7** [comment]
@anantprsd5  @isuruAb it has been over 30 days and this PR is pending for review!!!
Please review it.

**Comment by batbrain7** [comment]
@isuruAb please merge this approved PR

**Comment by batbrain7** [comment]
@isuruAb please merge this approved PR

**Comment by batbrain7** [comment]
@isuruAb please merge this approved PR!!

**Comment by batbrain7** [comment]
@isuruAb due to some errors at my side i am sending another pull request with these changes so please review and merge the new PR,
although the changes are the same as approved in this PR.

**Pull request closed without merging**